pkg/kgo/record_and_fetch.go

```diff
@@ -521,6 +522,19 @@ beforePartition:
 	}
 }
 
+// RecordsAll returns a Go native iterator that yields the records in a fetch.
+//
+// Similarly to RecordIter(), the errors should be inspected separately.
+func (fs Fetches) RecordsAll() iter.Seq[*Record] {
```

Comment:
I just really wish I didn't create the `Iter` method years before `Iter` was introduced into Go :(

Reply:
Actually that method is named `RecordIter`. Regardless, same sentence.

Reply:
To me, it's more annoying that fetches already has the `Records` method, tbf. But there is, of course, a historical truth there.

I've noticed the naming convention in Go's own std uses `All`, `Things` and `SomethingSeq` names, with a very subtle difference in the meaning, e.g. 

* `maps.All`, `maps.Keys`
* `bytes.SplitSeq`

I've picked `RecordsAll` because, this matches with "current" [Go docs][1]

> The iterator method on a collection type is conventionally named All, because it iterates a sequence of all the values in the collection. 

[1]: https://pkg.go.dev/iter#hdr-Naming_Conventions

Reply:
Ah, good justification, +1

Reply:
It may be worth adding a reference to this method in the docs on Iter, as well (basically, reference in both directions):

```
// You can also use [RecordsAll] for a native Go iterator.
```
or something.